### ISSUE TITLE
[opencv] build failed

### ISSUE THREAD

**Issue by twMr7** [Microsoft/vcpkg#4365]
`.\vcpkg install opencv[core,opengl,dnn,ovis,flann,sfm,contrib,cuda,ffmpeg,ipp,qt,vtk,openexr,tiff,png,jpeg,jasper,eigen]`

```
CMake Error at scripts/cmake/vcpkg_build_cmake.cmake:175 (message):
    Command failed: D:/RnD/vcpkg/downloads/tools/cmake-3.11.4-windows/cmake-3.11.4-win32-x86/bin/cmake.exe;--build;.;--config;Debug;--target;install;--;-v;-j1
    Working Directory: C:/RnD/vcpkg/buildtrees/opencv/x64-windows-dbg
    See logs for more information:
      C:\RnD\vcpkg\buildtrees\opencv\install-x64-windows-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/opencv/portfile.cmake:316 (vcpkg_install_cmake)
  scripts/ports.cmake:71 (include)

Error: Building package opencv:x64-windows failed with: BUILD_FAILED
```

The contents in install-x64-windows-dbg-err.log are as below.
```
ninja: error: 'C:/RnD/vcpkg/installed/x64-windows/lib/manual-link/OgreMain_d.lib', needed by 'bin/opencv_ovis341d.dll', missing and no known rule to make it
```

Obviously, it missed the debug build directory, but how to fix it?

**Comment by Voskrese** [comment]
```
    See logs for more information:
      C:\RnD\vcpkg\buildtrees\opencv\install-x64-windows-dbg-err.log
```

attach

**Comment by twMr7** [comment]
Here is the log file.
[install-x64-windows-dbg-err.log](https://github.com/Microsoft/vcpkg/files/2430893/install-x64-windows-dbg-err.log)

**Comment by Voskrese** [comment]
?
`dir C:/RnD/vcpkg/installed/x64-windows/bin/opencv_ovis*.dll`

**Comment by twMr7** [comment]
`ovis` feature should be fixed by PR #4845

**Comment by LarryIII** [comment]
@twMr7 Hello. Thanks for your feedback. These issues should be fixed. Could you please close this issue?
